### PR TITLE
Updating storage methane emission calculations

### PR DESCRIPTION
**Pull request by ltp23**
This PR updates the equation for calculating methane emission for slurry storage systems. This is done via the `calc_methane_emission_from_slurry_storage` method. 

The `_calc_arrhenius_exponent` and `_calc_volatile_solid_components` methods are helper methods that compute the Arrhenius exponent and the degradable and non-degradable volatile solid components respectively, which are used in the methane emission calculation. 

I also added unit tests for the changes I made.

### CODE REVIEW

**Review by varrma — APPROVED** [review]
Looks good Loi !!

File: RUFAS/routines/manure/constants/gas_emission_constants.py [hunk 1]
```diff
@@ -1,27 +1,30 @@
 class GasEmissionConstants:
     """Constants used in gas emission calculations."""
 
-    b1 = 1.0
-    """Rate correcting factor (unitless)."""
+    METHANE_EMISSION_COEFFICIENT = 24
+    """Methane emission coefficient, used in calculation of slurry storage methane emission (unitless)."""
```
Comment (ew3361zh) [inline RUFAS/routines/manure/constants/gas_emission_constants.py:5]: Is there another way to help explain what this is? I think the repeating of the name of variable in the definition could be changed to explain better what the coefficient is. Like for example the other two say "rate correcting factor for...". Is there something similar for this one? Or maybe it's just changing it to "The coefficient used in calculation of slurry storage methane emission"

**Review by ew3361zh — APPROVED** [review]
Nice work, Loi! I had one comment about the definition of a variable not being as clear as I think it could be but otherwise, this looks great and testing was working well.

**Review by JoeWaddell — APPROVED** [review]
This all looks great! Nice work.

**Pull request merged**